engine/renderer.py

```diff
@@ -3,7 +3,7 @@
 import math
 import time
 from engine.renderer_utils import out_dir, ray_aabb_intersection, inf, eps, \
-  intersect_sphere, sphere_aabb_intersect_motion, inside_taichi
+    intersect_sphere, sphere_aabb_intersect_motion, inside_taichi
```

Comment:
Why are these changed? If these changes are due to the code formatter, could you create a separate PR for these changes so that this PR gets easier to review?

Reply:
sure during commit I didn't notice that